src/acceleration/config/AccelerationConfiguration.cpp

```diff
@@ -149,10 +149,11 @@ void AccelerationConfiguration::xmlTagCallback(
     _config.relaxationFactor = callingTag.getDoubleAttributeValue(ATTR_VALUE);
   } else if (callingTag.getName() == TAG_DATA) {
     std::string dataName = callingTag.getStringAttributeValue(ATTR_NAME);
-    auto        success  = _uniqueDataNames.insert(dataName);
+    std::string meshName = callingTag.getStringAttributeValue(ATTR_MESH);
+    auto        success  = _uniqueDataAndMeshNames.insert(std::pair<std::string, std::string>(dataName, meshName));
```

Comment:
You could use `. emplace(dataName, meshName) ` here to remove the type boilerplate.

Reply:
Do you mean something like this?

```suggestion
    auto            ret = _uniqueDataAndMeshNames.emplace(dataName, meshName);
    auto        success = ret.second;
```

Reply:
And, of course, thanks for the hint!